ChangeLog.md

```diff
@@ -18,6 +18,15 @@ See docs/process.md for how version tagging works.
 Current Trunk
 -------------
 
+2.0.0: ??/??/????
+-----------------
+- First release that only supports the new upstream wasm backend (which has been
+  the default for a long time) and no longer supports the old fastcomp backend.
+  (#11319)
+- Store exceptions metadata in wasm memory instead of JS. This makes exception
+  handling almost 100% thread-safe. (#11518)
+
+
```

Comment:
Only one newline.    Maybe make this into a separate change that we can actually tag as 2.0.0?

Reply:
Sure, removed.